packages/govuk-frontend/src/govuk/helpers/_typography.scss

```diff
@@ -125,8 +149,14 @@
     $font-size: map-get($breakpoint-map, "font-size");
     $font-size-rem: govuk-px-to-rem($font-size);
 
-    $line-height: _govuk-line-height(
-      $line-height: if($override-line-height, $override-line-height, map-get($breakpoint-map, "line-height")),
+    // We give this variable a separate name to the $line-height param because
+    // otherwise the variable's value is redefined per loop and eventually
+    // _govuk-line-height breaks. We retain the param being called $line-height
+    // instead of this one because $line-height is consistent with the param
+    // naming for govuk-font, meaning it's easier for users to use them both
+    // with minimal friction.
```

Comment:
Can we simplify this sentence with something like this:

> This variable is separate from the $line-height param, as otherwise this variable will get redefined with each loop and eventually break  `_govuk-line-height`.
> 
> We continue to call the param $line-height to keep consistent naming with govuk-font, and make both easier to use.

Also, do can we give the name of the variable instead of calling it "this variable"?

Reply:
Another small go at this based on a brief Slack chat with @owenatgov 

> `$calculated-line-height` is a separate variable from the `$line-height` param, as otherwise the value would get redefined with each loop and eventually break  `_govuk-line-height`.
> 
> We continue to call the param $line-height to stay consistent with the naming with govuk-font, and make both easier to use.

Reply:
I'd like to suggest some light amendments:

>`$calculated-line-height` is a separate variable from `$line-height`, as otherwise the value would get redefined with each loop and eventually break `_govuk-line-height`.
>
>We continue to call the variable `$line-height` to stay consistent with the naming in govuk-font.

Why:

- This carries over from the original comment so it's my bad but it's probably confusing referring to "variable" and "param" in the same sentence when they mean the same thing
- It doesn't feel quite right to me to say that it makes both easier to use. It makes it so that users don't have to remember to use different variable names when swapping between the different mixins. I think just saying it's to keep it consistent does all the heavy lifting of the sentence.

Reply:
Yep looks good to me, much simpler! Thanks for helping me muddle through the terminology.

Reply:
DONE